pipelines/jobs/pipeline_job_template.groovy

```diff
@@ -59,6 +59,8 @@ pipelineJob("${BUILD_FOLDER}/${JOB_NAME}") {
     parameters {
```

Comment:
The no schedule behaviour was intentional in order to generate pipelines without times https://github.com/AdoptOpenJDK/openjdk-build/pull/2019. `ENABLE_PIPELINE_SCHEDULE` is set by default on jenkins

Reply:
Yes, it's fine to have no schedule. My point is, in the config triggers section "build periodically" will be checked, but the schedule will be empty. It's not functionally wrong, just a bit confusing maybe when I look at the job config, the schedule is enabled, but the textblock is empty. It would be cleaner, imo, to not have the schedule enabled.

Reply:
Ah I see what you mean. Yeah that would be easier to read, I'll do that

Reply:
Done

Reply:
@AdamBrousseau Ok so after playing around for a while on my fork of this repo, I haven't managed to get it generating without having the `build periodically` box checked. The reason for this is some flawed logic in how I am doing the `if...else` statement.

When `ENABLE_PIPELINE_SCHEDULE` is true, it sets the schedule to the `triggerSchedule` in the nightly config file (as expected). However, when it's false, it assigns the `@daily` tag which is obviously not what we want. Since this pr has been open for a long time already and it's not overly a big concern (the param in its current state is functional), I'm going to make the executive suggestion to leave it as is for now and revisit in the future if it becomes an issue

Reply:
FWM, thanks for trying.